Library/Homebrew/cmd/update-report.rb

```diff
@@ -235,20 +235,22 @@ def output_update_report
       EOS
     end
 
-    return if new_repository_version.blank?
+    return if new_tag.blank? || args.quiet?
 
     puts
-    ohai "Homebrew was updated to version #{new_repository_version}"
-    Settings.write "latesttag", new_repository_version
-    if new_repository_version.split(".").last == "0"
+    ohai "Homebrew was updated to version #{new_tag}"
+    puts <<~EOS
+      Changelogs for all Homebrew versions can be found at:
+        #{Formatter.url("https://github.com/Homebrew/brew/releases")}
+    EOS
+
+    new_major_version, new_minor_version = (new_tag.split(".")[0, 2]).map(&:to_i)
+    old_major_version, old_minor_version = (old_tag.split(".")[0, 2]).map(&:to_i) if old_tag.present?
+    if old_tag.blank? || (new_major_version > old_major_version) \
+        || (new_minor_version > old_minor_version)
       puts <<~EOS
         More detailed release notes are available on the Homebrew Blog:
-          #{Formatter.url("https://brew.sh/blog/#{new_repository_version}")}
-      EOS
-    elsif !args.quiet?
-      puts <<~EOS
-        The changelog can be found at:
-          #{Formatter.url("https://github.com/Homebrew/brew/releases/tag/#{new_repository_version}")}
+          #{Formatter.url("https://brew.sh/blog/")}
```

Comment:
```ruby
puts "More detailed release notes are available on the Homebrew Blog:"
new_tags.select { |t| t.end_with?(".0") }.each do |tag|
  puts "  #{Formatter.url("https://brew.sh/blog/#{tag}")}"
end
```

That way, we can list every major/minor version

Reply:
Is there a consensus on this? I prefer the less verbose generic reference to the main blog page. A user can click on from there to the specific version they are interested in.

Reply:
Also see https://github.com/Homebrew/brew/pull/13482#discussion_r913229761

Reply:
I think only output the latest version that's relevant (perhaps for each type e.g. blog and changeling).

Reply:
Reverted per https://github.com/Homebrew/brew/pull/13482#discussion_r913653140.